internal/pkg/secureboot/measure/internal/pcr/policy.go

```diff
@@ -0,0 +1,31 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package pcr
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+
+	"github.com/google/go-tpm/tpm2"
+)
+
+// CalculatePolicy calculates the policy hash for a given PCR value and PCR selection.
+func CalculatePolicy(pcrValue []byte, pcrSelection tpm2.TPMLPCRSelection) []byte {
```

Comment:
there is a slightly duplicate version of this in `internal/tpm2`, we could combine them, but it's also fine to keep as is.

Reply:
I will take a look

Reply:
it feels like `tpm2` version is slightly different, so I don't feel confident unifying them, even though I see some overlap

Reply:
ahh yes, one passes in existing digest, other starts with zero